test/sdf/shapes_world.sdf

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" ?>
```

Comment:
I didn't see a test that uses this new file.

Reply:
yeah I didn't write a test; this is more of a demo. the `test/sdf/shapes.world` is kind of a disaster to actually use because all the shapes are at the same pose. is it enough to add an `ign sdf --check` test for it?

Reply:
testing with `ign sdf --check` as of 09f0658e335f479b149a67e7dbb19feb785fd4b2; let me know if you want a better test than that

Reply:
Sure. I'd be okay with just putting the file in the examples directory as well. I was just confused to see it in `/test/sdf` without a corresponding test.